src/lib.rs

```diff
@@ -7,7 +7,7 @@
 extern crate test;
 
 /// An iterator that runs all at once
-pub trait Traversal<T> {
+pub trait Traversal<T> : Sized {
```

Comment:
Can you change this to `Traversal<T>: Sized` with no space between the `>` and `:`?